platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/config/MerkleDbConfig.java

```diff
@@ -81,6 +85,10 @@ public record MerkleDbConfig(
         @Positive @ConfigProperty(defaultValue = "16777216") int iteratorInputBufferBytes,
         @ConfigProperty(defaultValue = "false") boolean reconnectKeyLeakMitigationEnabled,
         @ConfigProperty(defaultValue = "false") boolean indexRebuildingEnforced,
+        @ConfigProperty(
+                        defaultValue =
+                                "ScheduleService.SCHEDULED_COUNTS,ScheduleService.SCHEDULED_ORDERS,ScheduleService.SCHEDULED_USAGES,ScheduleService.SCHEDULES_BY_ID,ScheduleService.SCHEDULE_ID_BY_EQUALITY,TokenService.PENDING_AIRDROPS")
```

Comment:
Let's exclude `TokenService.PENDING_AIRDROPS` from the default configuration (pending confirmation we don't need to repair the map in mainnet). For testnet, it can be added to settings.txt.

Reply:
Done